packages/react-components/components/LoadingLabel.tsx

```diff
@@ -0,0 +1,70 @@
+import colors from '@celo/react-components/styles/colors'
```

Comment:
A new reusable that's basically a label immediately followed by a value or a spinner if value is falsy